web/src/compose_reply.ts

```diff
@@ -267,6 +282,14 @@ export function quote_message(opts: {
 
         compose_ui.replace_syntax(quoting_placeholder, content, $textarea);
         compose_ui.autosize_textarea($textarea);
+
+        if (opts.forward_message) {
+            const select_recipient_widget: tippy.ReferenceElement | undefined = $(
+                "#compose_select_recipient_widget",
+            )[0];
+            assert(select_recipient_widget !== undefined);
```

Comment:
I don't think we can assert this is visible; since time has passed, the user might have done something that closes the tooltip. You want to instead just do an `if` statement and do nothing if it's not present.

Reply:
Yeah true, fixed it now!